peer/peer.go

```diff
@@ -1861,12 +1861,12 @@ func (p *Peer) AssociateConnection(conn net.Conn) {
 		p.na = na
 	}
 
-	go func() {
-		if err := p.start(); err != nil {
-			log.Debugf("Cannot start peer %v: %v", p, err)
-			p.Disconnect()
+	go func(pr *Peer) {
```

Comment:
Please call the parameter `peer`.  Nowhere else in the codebase refers to a peer as pr.